pkgs/development/python-modules/langgraph-checkpoint-postgres/default.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  langgraph-checkpoint,
+  psycopg,
+  langgraph-sdk,
+  poetry-core,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "langgraph-checkpoint-postgres";
+  version = "1.0.3";
+  pyproject = true;
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchFromGitHub {
+    owner = "langchain-ai";
+    repo = "langgraph";
+    rev = "refs/tags/checkpointpostgres==${version}";
+    hash = "sha256-U7Bymo+Nj82kwjxN33W2MT10jv+lioZUxIKUt8Yxh/s=";
+  };
+
+  sourceRoot = "${src.name}/libs/checkpoint-postgres";
+
+  build-system = [ poetry-core ];
+
+  dependencies = [
+    langgraph-checkpoint
+    psycopg
```

```suggestion
    langgraph-checkpoint
    orjson
    psycopg
```
https://github.com/langchain-ai/langgraph/blob/d98eec6e91d849ef33c7b2d33e821a3ecdf4431b/libs/checkpoint-postgres/pyproject.toml#L11-L15

---

Fixed.